CLAUDE.md

```
@@ -0,0 +1,106 @@
+pre-commit run -a                  # run all hooks
+black .                            # format (line-length: 119)
+isort .                            # sort imports (black profile)
+flake8 .                           # lint
+
+# Celery
+celery -A connectid.celery_app worker -B -l INFO
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)